GVRf/Framework/src/org/gearvrf/scene_objects/GVRTextViewSceneObject.java

```diff
@@ -335,12 +369,56 @@ public int getGravity() {
         return mTextView.getGravity();
     }
 
+    /**
+     * Set the refresh frequency of this scene object.
+     * 
+     * @param frequency
+     *            The refresh frequency of this TextViewSceneObject.
+     */
+    public void setRefreshFrequency(IntervalFrequency frequency) {
```

Comment:
I believe you tested these setty()/getty() methods from a sample app. Your decision, but it would be helpful to submit the updated sample app, too.